web/src/lib/components/pages.tsx

```diff
@@ -42,30 +41,15 @@ const URLS: PageUrls = {
   NOTFOUND: '/not-found',
 };
 
-const ROBOT_TALK = {
-  home: [
-    <p>Greetings human!</p>,
-    <p>My name is M.A.R.S. and I am a learning robot.</p>,
-    <p>Right now, I am learning to speak like a human.</p>,
-    <p>But. . . it's so hard!</p>,
-    <p>Can you help me learn?</p>,
-    <p>All I need is for you to read to me. :)</p>,
-    <p>Please click on the heart below to get started teaching me.</p>,
-  ],
-};
```

Comment:
MARS shall forever be with us in our hearts and gits. Unfortunately we don't yet have reliable dead code removal, so I'm throwing everything out that isn't used (also want to add an ESLint rule for that, but maybe we should talk about that at some future date).

Reply:
I believe [UglifyjsWebpackPlugin](https://webpack.js.org/plugins/uglifyjs-webpack-plugin/) already removes dead code from the final build. (since that plugin internally uses [*uglify-es* package](https://github.com/mishoo/UglifyJS2/tree/harmony#compress-options) which has "dead_code" option (dead code elimination) turned on by default)

Reply:
Indeed! Thought I tested if it was part of the bundle, but I probably only compared it to the gulp build (where it indeed was part of the bundle). Nice!
Though I'd still stay by my point of not leaving unused code in.